.gitignore

```diff
@@ -296,3 +296,5 @@ Doc
 appsettings.local.json
 Docs/.DS_Store
 .DS_Store
+
+Tools/Cli-LoRa-Device-Provisioning/settings\.local\.json
```

Comment:
We should generalize local settings file names to avoid adding multiple .gitignore files. I can see there is only already to `appsetings.local.json`.
My suggestion is to either use the same name or change the gitignore rule to *.local.json

Reply:
I am happy to switch. 
Should I change both files to appsettings.json and appsettings.local.json? Makes sense to have consistency.
Then undo the .gitignore changes...

Reply:
I think that is a good approach, specially because Azure Functions takes `appsettings.local.json` by default.

Reply:
Done!